src/witan/send/adroddiad/vega_specs/lines.clj

```diff
@@ -103,10 +105,10 @@
      :config   {:legend {:titleFontSize 20 :labelFontSize 14 :labelLimit 0}
                 :axisX  {:titleFontSize 16 :labelFontSize 12}
                 :axisY  {:titleFontSize 16 :labelFontSize 12}}
-     :encoding {:x {:field x :title x-title :type "temporal"
-                    :scale {:domain x-scale :x-xero false}
-                    ;; :axis {:format ["%Y"] :tickCount {:interval "month" :step 12}}
-                    }
+     :encoding {:x     {:field x :title x-title :type "temporal"
+                        :scale {:domain x-scale :x-xero false}
+                        ;; :axis {:format ["%Y"] :tickCount {:interval "month" :step 12}}
+                        }
```

Comment:
why whitespace? (and below)

Reply:
I ran `clojure-align` and it pushed the `:x` vals right to line up with the vals of the `:color` keys added in PR#104.